contracts/ERC5018.sol

```diff
@@ -3,40 +3,61 @@ pragma solidity ^0.8.0;
 
 import "./IERC5018.sol";
 import "./LargeStorageManager.sol";
+import "./BlobStorageManager.sol";
 
-contract ERC5018 is IERC5018, LargeStorageManager {
-    address public owner;
+contract ERC5018 is IERC5018, LargeStorageManager, BlobStorageManager  {
 
-    constructor(uint8 slotLimit) LargeStorageManager(slotLimit) {
-        owner = msg.sender;
+    enum FileMode {
+        Uninitialized,
+        CallData,
```

Comment:
CallData is not a storage place, and what we used here is on-chain slot or code section, so a better name would be OnChain?